### ISSUE TITLE
run checks against index.slim ?

### ISSUE THREAD

**Issue by shyukri** [addyosmani/a11y#70]
I'm trying to run checks for pages using https://github.com/slim-template/slim
Is it possible?

`find . -name "index.slim" | xargs a11y`

```
Command failed: /usr/local/bin/phantomjs /usr/local/lib/node_modules/a11y/audits.js {"delay":1,"url":"file:///~/app/views/.../index.slim","width":1024,"height":768} --ignore-ssl-errors=true --ssl-protocol=tlsv1 --local-to-remote-url-access=true
Couldn't load url: "file:///~/app/views/.../index.slim"
```
